scripts/zones/Mhaura/npcs/Tonasav.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Selbina (249)
+--  NPC: Tonasav
+-- Type: ROV NPC
+-----------------------------------
+require("scripts/globals/missions")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    if player:getCurrentMission(ROV) == dsp.mission.id.rov.THE_BEGINNING and player:getVar("ROV") == 0 then
```

Comment:
If thats a mission status variable, we have an existing convention for those. ZilartStatus PromathiaStatus etc. etc.. so RhapsodiesStatus probably. RovStatus maybe? The mission category seems to be using ROV already, so whichever.